labelmerge/workflow/scripts/label_split.py

```diff
@@ -76,7 +76,7 @@ def label_split(atlas_path, output_dir, smk_wildcards, bids_dir):
             label_desc = atlas_metadata[atlas_metadata["Index"] == int(label)][
                 "BIDS Name"
             ].values[0]
-        except:
+        except:  # noqa: E722
```

Comment:
I think this check is actually valid -- we should figure out what gets raised if the label is missing, catch that exception specifically, and raise the `ValueError` (we may also want to define a `MetadataError` or something) from that exception.

Reply:
Fair - I removed this `noqa` and added in the IndexError (due to `values[0]` returning an empty list if `int(label)` does not exist in the dataframe).

I tried to define a MetadataError but was running into a few issues. Can look into defining one in a future commit / separate PR.